doc/sphinx/source/howto/update.rst

```diff
@@ -0,0 +1,48 @@
+.. update.rst
+
+###########################
+Make a Stateful Computation
+###########################
+
+In this section, we show how to make a stateful computation from
+nGraphs stateless operations. The basic idea is that any computation
```

Comment:
Note:  My guess is that this kind of picks up from where @Krovatkin and I left off working on documentation around op fusion.  How does this intro sentence from [here](https://github.com/NervanaSystems/ngraph/blob/leona/21_Mar/doc/sphinx/source/howto/fuse.rst) work?  

> In working with a function graph, there are many ways to describe what happens when we need to do something with the ops (AKA "operational functions") from that graph.   

Then... 

> This section describes how to make a *stateful* computation from the *statelessness* of nGraph ops.  

?

Reply:
They are different things.